flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+	"only-arches": [ "x86_64" ]
```

Comment:
this is really kind unfortunate.

Reply:
so why are you limiting to x86_64?

Reply:
I have only tested it on `x86_64` and currenly only provide the custom emulator core on 64-bit for Linux. I'm not comfortable releasing it for all architectures yet since I haven't tested them. I will add more as I test on more hardware. All of my current users on Linux use x86_64.

Reply:
aarch64 is 64-bits linux too.

Reply:
and retroarch is available on aarch64.

Reply:
Yes, but I haven't tested it to verify that it works as expected, so I won't release it for that arch until I can verify I have done endianness detection correctly everywhere and aren't accidentally relying on x86 somewhere.